### PR TITLE
Cleanup and fix build

### PR DESCRIPTION
**Pull request by BachToTheFuture**
Oops. Forgot about screenshots. Will add them to the readme.

### CODE REVIEW

File: Screensaver/BinaryClockSS.rdef [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+resource app_version {
+	major  = 0,
+	middle = 7,
+	minor  = 0,
+
+	variety = B_APPV_DEVELOPMENT,
+	internal = 0,
+
+	short_info = "Binary clock screen saver",
+	long_info = "Written by David Enderson, http://neelix.festing.org,\
```
Comment (humdingerb) [inline Screensaver/BinaryClockSS.rdef]: Same as comment on the main app's rdef applies.

**Review by humdingerb — CHANGES_REQUESTED** [review]
Move the ReadMe.md from the subfolder up into the top folder.

**Comment by bach5000** [comment]
@humdingerb Which README should I put in the top directory? Each subdirectory has a different README...
Should I merge the README contents together or something?

**Comment by humdingerb** [comment]
Just use the one now in Application.

**Comment by scottmc** [comment]
Please remove any broken weblinks in readmes, and old email addresses, etc.  Issues should now be reported here now.  Also please add a LICENSE file to the root, you can create this by copying the GPL2 license from and then adding the authors info into it.  Author stated that the code is GPL in BinaryClockSS.cpp, judging by the date of the code I'd assume this would mean GPL2.

**Pull request merged**